src/Widgets/TaskRow.vala

```diff
@@ -334,7 +336,11 @@ public class Tasks.TaskRow : Gtk.ListBoxRow {
     }
 
     public void update_request () {
-        Tasks.Application.set_task_color (source, check);
+        if (is_scheduled_view && tint_color != null) {
+            Tasks.Application.set_task_color_from_string (tint_color, check);
```

Comment:
This doesn't actually do anything since the only place this is called sets the check to the accent color (which it already is by default)